backend/coins/btc/handlers/handlers.go

```diff
@@ -230,6 +231,10 @@ func (handlers *Handlers) getAccountTransactions(_ *http.Request) (interface{},
 	}
 	result.Transactions = []Transaction{}
 	for _, txInfo := range txs {
+		if txInfo.FeeIsDifferentUnit && big.NewInt(0).Cmp(txInfo.Amount.BigInt()) == 0 {
```

Comment:
`FeeIsDifferentUnit` does not imply ERC-20 token necessarily. 

I think it's better to move this logic to `eth/account.go:Transactions()` (or to wherever the transactions are populated), where you can also directly check if it is an ERC-20 token. Generally/ideally, the HTTP handlers should not contain any logic but just parse input and assemble the JSON output.

Reply:
As discussed, I didn't change this, to avoid modifying the persisted transactions, which could have consequences on e.g. balance computing. The code is only for frontend use anyway.

Reply:
The part about `FeeIsDifferentUnit does not imply ERC-20 token necessarily.` still applies though :) Best to make a new field in the tx-info struct `IsERC20` and set/use it accordingly